changelog/796.doc.rst

```diff
@@ -0,0 +1 @@
+Add narrative documentation on ionization state functionality.
```

Comment:
Didn't know towncrier would handle two snippets per PR. Seems to work nicely, though. TIL, thanks! :+1: